cost-analyzer/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "1.92.0"
+appVersion: "1.97.0"
 description: A Helm chart that sets up Kubecost, Prometheus, and Grafana to monitor
   cloud costs.
 name: cost-analyzer
-version: "1.92.0"
+version: "1.97.0"
```

Comment:
Would you omit the appVersion update? I think we'll hit a merge conflict between this and https://github.com/kubecost/cost-analyzer-helm-chart/pull/1793. https://github.com/kubecost/cost-analyzer-helm-chart/pull/1793 is more tightly scoped to the appVersion update so I'd prefer to do it there.

Reply:
Good point, will do!